ckanext/fluent/validators.py

```diff
@@ -155,9 +156,13 @@ def validator(key, data, errors, context):
         if output is None:
             return
 
-        for lang in output:
-            del extras[prefix + lang]
-        data[key] = json.dumps(output, ensure_ascii=False)
+        # Return None if alle values in json are None or "" (state = deleted in package_extra)
+        if all(v is None or v == "" for v in output.values()):
```

Comment:
Komt hij ook terecht in if statement #3 als de field_name de extensie _translated krijgt? Ik heb de indruk dat de gevalideerde value een< multilingual dict> en/of een <JSON encoded version of a multilingual dict>.

En dan komt hij in IF statement <# 1 or 2. dict or JSON encoded string> terecht en daar zit the data[key] = None nog niet in verwerkt...